aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -374,17 +389,24 @@ pub fn extension_error(message: impl ToString) -> PartialVMError {
 mod test {
     use super::*;
     use aptos_language_e2e_tests::data_store::FakeDataStore;
+    use aptos_types::state_store::state_key::StateKey;
     use claims::{assert_err, assert_ok};
     use once_cell::sync::Lazy;
 
+    pub fn serialize(value: &u128) -> Vec<u8> {
```

Comment:
nit: serialization function already exists (actually in `delta_change_set.rs`, checked now)

Reply:
Done.